python/res/simulator/batch_simulator_context.py

```diff
@@ -78,6 +78,7 @@ def results(self):
         nodes = [ EnkfNode(self.res_config.ensemble_config[key]) for key in self.result_keys ]
         for sim_id in range(len(self)):
             node_id = NodeId( 0, sim_id)
+
```

Comment:
Please revert white space fixes. This PR is touching enough files as is.

Reply:
fixed